tests/puppeteer/pages/BO/catalog/products/add.js

```diff
@@ -295,4 +297,12 @@ module.exports = class AddProduct extends BOBasePage {
     await this.goToFormStep(5);
     return this.getAttributeContent(this.friendlyUrlInput, 'value');
   }
+
+  /**
+   * Get online product status
+   * @returns {Promise<boolean>}
+   */
+  async getOnlineButtonStatus() {
```

Comment:
```suggestion
  getOnlineButtonStatus() {
```
No need to async if there is no await instruction